hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -301,6 +301,15 @@ public void printReport() {
     messages.forEach(print);
   }
 
+  public void print(String s){
```

Comment:
Latest patch looks good. Adding few minor comments:

Please add javadoc and change the parameter name 's' to 'msg', just to improve more readable.

Reply:
Added javadoc, changed 's' to 'msg'.